packages/core/src/App/Containers/show.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LocationContext } from './location-context';
+
+export const Eu = ({ children }) => {
```

Comment:
I think we don't need these components in deriv-app? @reza-fs

Reply:
Agreed.